libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -206,6 +206,8 @@ void AP_AHRS_NavEKF::update(bool skip_ins_update)
             shortname = "EKF2";
             break;
 #endif
+        default:
+            break;
```

Comment:
The enumeration values should not be present when the EKF is compiled out.  That will make this `default` unnecessary.